lib/generation/generators/attribute-helper/pb_box_decoration_gen_helper.dart

```diff
@@ -13,6 +13,7 @@ class PBBoxDecorationHelper extends PBAttributesHelper {
     if (source is InheritedContainer) {
       final buffer = StringBuffer();
       buffer.write('decoration: BoxDecoration(');
+      buffer.write(PBColorGenHelper().generate(source, generatorContext));
```

Comment:
Line 16 makes the if statement on line 18 not necessary. Otherwise, we get duplicate `color` properties 🤔 . Here is an example:

![Screen Shot 2021-05-24 at 10 58 25 AM](https://user-images.githubusercontent.com/42812006/119382217-f824e200-bc7e-11eb-95e9-72658c284f7a.png)